src/slacko.ml

```diff
@@ -834,7 +841,11 @@ let string_of_presence = function
   | Away -> "away"
 
 (* Slacko API helper methods *)
-let token_of_string = identity
+let make_session ?(base_url=default_base_url) token = {
```

Comment:
How about `start_session`? This makes more sense in a Session context, but maybe `make_` makes more sense in an OCaml context (with `Make` functors). Or maybe `create_session`?